docs/book/v2/quick-start.md

```diff
@@ -0,0 +1,117 @@
+# Quick Start
```

Comment:
The quick start must contain a complete and comprehensible example for the correct use of this component. This means to add also paths and filenames and concrete examples instead of "foo", "bar", "some" and so on.

For example:

> Define a route for a request handler in a module configuration file, e.g. `module/Album/config/module.config.php`:
> ```php
> use Album\Middleware\AlbumListHandler;
> use Laminas\Mvc\Middleware\PipeSpec;
> use Laminas\Router\Http\Literal;
> 
> return [
>     'router' => [
>         'routes' => [
>             'album' => [
>                 'type' => Literal::class,
>                 'options' => [
>                     'route' => '/album',
>                     'defaults' => [
>                         'controller' => PipeSpec::class,
>                         'middleware' => AlbumListHandler::class,
>                     ],
>                 ],
>             ],
>         ],
>     ],
> ];
> ```

In this way we avoid one of the main problems of documentation.

Reply:
okay, I tried to use real-world examples... more or less.
Is that what you were aiming for?